js/src/components/google-combo-account-card/index.scss

```
@@ -0,0 +1,10 @@
+.google-combo-account-card {
+	.gla-account-card__description {
+		.components-checkbox-control {
+			display: inline-block;
+		}
+	}
+	p {
```

```suggestion

	p {
```
Leave an empty line between the selectors please.